JSDemos/Demos/PivotGrid/DrillDown/description.md

```diff
@@ -1 +1,5 @@
-Drill-down support within the PivotGrid component allows you to&nbsp;retrieve the list of&nbsp;records used to&nbsp;calculate a&nbsp;specific summary value. To&nbsp;see drill downs in&nbsp;action, click a&nbsp;summary cell and view the records that formed the basis of&nbsp;the summary value.
\ No newline at end of file
+PivotGrid supports the drill down operation that allows users to retrieve the list of facts used to calculate a specific summary value. 
```

Comment:
I presume users want to _display_ the list of facts. Also, "fact" is a term that may be unfamiliar for novices. Thus the suggestion is
```suggestion
PivotGrid supports the drill down operation that allows users to display the list of [facts](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Summary_Values) (records) used to calculate a specific summary value. 
```

Reply:
The thing is that this method only returns collected data but does not display it anywhere. It is up to user to decide how to use it later.

About facts - I tend to agree.